### ISSUE TITLE
Preview/Next-Links right next to the navigation-dots

### ISSUE THREAD

**Issue by ursbrauchli** [stevenwanderski/bxslider-4#876]
Not exactly an issue, but something to consider.

I had to place the prev/next links right next to the navigation-dots:

![screenshot](https://cloud.githubusercontent.com/assets/12100878/8775559/9eda2f1c-2ee9-11e5-9707-ab1bd92b0acb.png)

The original script puts them in arbitrary places, so I used the onSliderLoad-function to place them where it need them:

onSliderLoad: function(){
		  				var prev = $('.slider-prev').detach();
		  				prev.prependTo('.bx-pager');
		  				var next = $('.slider-next').detach();
		  				next.appendTo('.bx-pager');
		  				$('a.bx-prev').removeClass('active');
		  				$('a.bx-pager-link').first().addClass('active');
		  			}

But with the prev/next links within the bx-controls -> bx-pager div, the selected index stopped working correctly and added an «active» class to the prev-link on load and skipped the last dot...
Solution was to change the updatePagerActive function like this:

// remove all pager active classes
slider.pagerEl.find('a.bx-pager-link').removeClass('active');
// apply the active class for all pagers
slider.pagerEl.each(function(i, el) { $(el).find('a.bx-pager-link').eq(slideIndex).addClass('active'); });

I just added the bx-pager-link class to the a-tag...

Maybe this helps somebody
